cloud-startup.sh

```diff
@@ -20,28 +31,86 @@ done
 
 echo 'Database connection successful'
 
-# Run database migrations
+# Create database if it doesn't exist
```

Comment:
In the local operation check, if the database does not exist, the mysql docker image side confirms the behavior to automatically create the database. (I believe there is an official explanation for this)
Is this code required?

Reply:
I think this was added for use with AWS cloud installation with RDS. 
I guess it may be possible to remove it, however, is there any benefit to removing it? If the db exists, then it has no effect. If it does not exist, it creates one.

Reply:
As you mentioned, the configuration uses AWS RDS, not a mysql docker image. Therefore, the initialization functionality provided by the docker image version is not provided.
-> approved.